utils/utils.go

```diff
@@ -350,6 +350,26 @@ func GetAbsPath(baseDir, path string) string {
 	return absPath
 }
 
+// ResolveHomeDir resolves prefix of the given path, if present, to the user's home directory and returns it
+func ResolveHomeDir(filePath string) (string, error) {
+	if filePath == "" {
+		return "", nil
+	}
+	homeDirPrefixes := []string{"~/", "$HOME/"}
```

Comment:
AFAIK in windows this variable `$HOME` is called `Userprofile` and this logic might not be able to handle it. Should we avoid this part ?

Reply:
Yes you are correct, I'll remove support for `$HOME` and support only `~/`